auditing/meilisearch.go

```diff
@@ -513,3 +518,40 @@ func indexName(prefix string, i Interval) string {
 	indexName := prefix + "-" + time.Now().Format(timeFormat)
 	return indexName
 }
+
+func isIndexRelevantForSearchRange(indexName string, from, to time.Time) bool {
+	intervalRe := regexp.MustCompile("\\d\\d\\d\\d-\\d\\d(-\\d\\d(_\\d\\d)?)?")
```

Comment:
i am not a huge fan of regex, but if it is not possible to implement it without, this should be made `const`